soc/realtek/ec/rts5912/Kconfig

```
@@ -29,4 +30,12 @@ config RTS5912_DEBUG_SWJ
 	help
 	  Enables the serial wire JTAG connection on the RTS5912 EC.
 
+config SOC_RTS5912_ULPM
+	bool "Realtek RTS5912 ULPM (Ultra Low Power Mode)"
+	default y
+	depends on DT_HAS_REALTEK_RTS5912_ULPM_ENABLED
+	select POWEROFF
```

```suggestion
	select HAS_POWEROFF
```
Users select POWEROFF if they want it not a soc

---

done, thanks.